linux-cachyos-autofdo/PKGBUILD

```diff
@@ -259,6 +259,8 @@ fi
 case "$_cpusched" in
     cachyos|bore|rt-bore|hardened) # CachyOS Scheduler (BORE)
         source+=("${_patchsource}/sched/0001-bore-cachy.patch");;&
+    rt-bore) ## RT with BORE Scheduler
```

Comment:
Can you add one for `rt` and move it down to keep it alphabetically ordered?

Reply:
I fixed it!